oio/xcute/server.py

```diff
@@ -73,10 +73,9 @@ def on_job_list(self, req):
             if job_class is None:
                 return HTTPBadRequest(UnknownJobTypeException.message)
 
-            job = job_class(self.conf, logger=self.logger)
-            job_config = data.get('config')
             # TODO: use lock
-            job_config, lock = job.sanitize_config(job_config)
+            job_config, lock = job_class.sanitize_config(
+                data.get('config') or dict())
```

Comment:
`data.get('config', dict)`?

Reply:
```
>>> mydict = dict(mykey=None)
>>> mydict.get('mykey')
>>> mydict.get('mykey', dict())
>>> mydict.get('mykey') or dict()
{}
```

Reply:
Why would you have None though?

Reply:
If you can have None, you may as well check that you have the right type